onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.py

```diff
@@ -60,3 +60,86 @@
 
 # pylint: disable=wrong-import-position, disable=unused-import, disable=wildcard-import
 from _kernel_explorer import *  # noqa
+
+"""Benchmark Reporter"""
+
+from abc import abstractmethod
+from contextlib import contextmanager
+from dataclasses import dataclass
+
+
+@dataclass
+class InstanceStatus:
```

Comment:
> class InstanceStatus

Naming: Would names like `MetricBase` and `ComputeMetric`, `BandwidthMetric` be more descriptive? Right now, `InstanceStatus` tells me nothing about what this class represents.

Reply:
Done.